spi_nand_flash/src/dhara_glue.c

```diff
@@ -154,6 +178,13 @@ int dhara_nand_prog(const struct dhara_nand *n, dhara_page_t p, const uint8_t *d
         return -1;
     }
 
+#if CONFIG_NAND_FLASH_VERIFY_WRITE
+    ret = s_verify_write(dev, data, 0, dev->page_size);
+    if (ret != ESP_OK) {
+        ESP_LOGE(TAG, "%s: prog page=%"PRIu32" write verification failed", __func__, p);
+    }
```

Comment:
Technically we also have to verify the used marker here

Reply:
I mean that a few lines above, we write two parts:

```c++
    ESP_GOTO_ON_ERROR(spi_nand_program_load(dev->config.device_handle, data, 0, dev->page_size),
                      fail, TAG, "");
    ESP_GOTO_ON_ERROR(spi_nand_program_load(dev->config.device_handle, (uint8_t *)&used_marker,
                                            dev->page_size + 2, 2),
```

This `s_verify_write` call checks that the first part (from 0 to page_size) is written correctly. We should call s_verify_write second time to check that the used page marker (page_size ... page_size + 2) is also written correctly.

Reply:
Oh, my mistake—I misunderstood. Yes, it should verify the markers. I’ll update it. Thank you!